packages/plugin-hardhat/test/beacon-timeout.js

```diff
@@ -0,0 +1,82 @@
+const test = require('ava');
```

Comment:
I don't think it's necessary to have a full test suite for all three kinds of proxies. What do you think? Is there a particular reason we need to test all three?